### DISCUSSION TITLE
[Question]: Any way to access the number of alerts/detections pending review in Home Assistant?

### DISCUSSION THREAD

**Discussion by PHLAK** [blakeblackshear/frigate#12890] [Ask A Question] [answered]
Is there any way to access the number of alerts/detections pending review in Home Assistant?

I would love to display these on my dashboard and/or use them in automations.

**Comment by hawkeye217** [comment, marked as answer]
Not currently. But it shouldn't be a difficult thing to add. Feel free to file a [feature request](https://github.com/blakeblackshear/frigate/issues/new/choose).

**Reply by PHLAK** [reply]
Thanks, [done](https://github.com/blakeblackshear/frigate/issues/12907)!
